config/locales/en.yml

```diff
@@ -93,6 +94,8 @@ en:
       receiving_organisation_reference:
         hint: For example, GB-COH-12345. To lookup codes or for more infomation see <a href="http://org-id.guide/" target="_blank" class="govuk-link">the organisation finder service (Opens in new window)</a>.
         label: IATI Reference (optional)
+      update:
+        success: Planned disbursement subsequently updated
```

Comment:
Why is this text "subsequently updated"? The other entities don't have this wording, is it a special thing for planned disbursements?

Reply:
Typo!